bin/snyk_tests.sh

```diff
@@ -22,13 +24,56 @@ source bin/common/check_returncode.sh
 # authenticate snyk CLI locally
 run_and_check_returncode "snyk auth $SNYK_AUTHENTICATION_TOKEN"
 
-# Iac run limited to 20 runs a month. TBD Discuss with Jon Velapondi.
-# echo "Scanning Infrastructure as code with Snyk for potential vulnerabilities..."
-# snyk iac test || true
-
-# scan orca website
+# Scan orca website
 echo "Scanning open-source code with Snyk for potential vulnerabilities..."
 cd website
 snyk test -d --severity-threshold=high || true
+cd -
+
+function run_snyk_tests() {
+  echo
+  echo "Running snyk test in $1"
+  echo
+
+  cd $1
+  # remove shared library from the requirement files since snyk cannot scan through it
+  grep -v '../../shared_libraries' requirements-dev.txt >> test-requirements-dev.txt
+  grep -v '../../shared_libraries' requirements.txt >> test-requirements.txt
+  snyk test --severity-threshold=high --file=test-requirements-dev.txt --package-manager=pip || true
+  snyk test --severity-threshold=high --file=test-requirements.txt --package-manager=pip || true
+  # remove test files
+  rm test-requirements*.txt
+}
+export -f run_snyk_tests
+
+function run_snyk_tests_others() {
```

Comment:
Looks good. What do you think about renaming the original function? Something like "run_snyk_tests_lambda" to compliment your "run_snyk_tests_other"?

Reply:
updated